build/isc-bind9/build-911.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+#
+# Copyright 2017 OmniTI Computer Consulting, Inc.  All rights reserved.
+# Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/functions.sh
+
+PROG=bind
+VER=9.11.22
+PKG=network/bind-911
+SUMMARY="ISC BIND DNS Server & Tools"
+DESC="Server & Client Utilities for DNS"
+
+RUN_DEPENDS_IPS="
+    ooce/network/bind-common
+"
+
+set_arch 64
+
+SKIP_LICENCES="*"
+
+MAJVER=${VER%.*}            # M.m
+sMAJVER=${MAJVER//./}       # Mm
+PATCHDIR=patches-$sMAJVER
+
+OPREFIX=$PREFIX
+PREFIX+=/$PROG-$MAJVER
+
+HARDLINK_TARGETS="
+    opt/ooce/bind-9.11/bin/isc-config.sh
+    opt/ooce/bind-9.11/sbin/named
+    opt/ooce/bind-9.11/share/man/man1/isc-config.sh.1
+    opt/ooce/bind-9.11/share/man/man3/lwres_resutil.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_buffer.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_config.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_context.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_gethostent.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_getaddrinfo.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_getipnode.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_gabn.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_gnba.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_hstrerror.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_packet.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_inetntop.3
+    opt/ooce/bind-9.11/share/man/man3/lwres_noop.3
+"
+
+XFORM_ARGS="
+    -DPREFIX=${PREFIX#/}
+    -DOPREFIX=${OPREFIX#/}
+    -DPROG=$PROG
+    -DVERSION=$MAJVER
+    -DsVERSION=$sMAJVER
+    -DUSER=named -DGROUP=named
+"
+
+CONFIGURE_OPTS="
+    --prefix=/$PREFIX
+    --bindir=/$PREFIX/bin
+    --sbindir=/$PREFIX/sbin
+    --libdir=/$PREFIX/lib
+    --includedir=/$PREFIX/include
+    --sysconfdir=/etc/$PREFIX
+    --localstatedir=/var/$PREFIX
+    --with-libtool
+    --with-openssl
+    --enable-threads=yes
+    --enable-devpoll=yes
+    --enable-fixed-rrset
+    --disable-getifaddrs
+    --enable-shared
+    --disable-static
+    --without-python
+    --with-zlib=/usr
+"
+
+init
+download_source $PROG $PROG $VER
+patch_source
+xform files/named.conf-template > $TMPDIR/named-$sMAJVER.conf
+prep_build
+run_autoreconf -fi
+build
+strip_install
+xform files/named-template.xml > $TMPDIR/named-$sMAJVER.xml
+install_smf network/dns named-$sMAJVER.xml
```

Comment:
We're moving all SMF methods/manifests from extra into the ooce namespace
```suggestion
install_smf ooce named-$sMAJVER.xml
```

Reply:
this is done, and is also now similar to nginx and apache, as discussed